python/cudf/cudf/_lib/cpp/stream_compaction.pxd

```diff
@@ -45,3 +47,10 @@ cdef extern from "cudf/stream_compaction.hpp" namespace "cudf" \
         duplicate_keep_option keep,
         null_equality nulls_equal,
     ) except +
+
+    cdef unique_ptr[column] distinct_indices(
+        table_view input,
+        duplicate_keep_option keep,
+        null_equality nulls_equal,
+        nan_equality nans_equal,
+    )
```

Comment:
`except +` missing. is this the intended behaviour?

Reply:
Related docs:
https://cython.readthedocs.io/en/latest/src/userguide/wrapping_CPlusPlus.html#exceptions
https://cython.readthedocs.io/en/latest/src/userguide/wrapping_CPlusPlus.html#add-public-attributes

Reply:
It is not, thanks!
```suggestion
    ) except +
```